taxonomy-editor-frontend/src/pages/allentries/EntriesList.jsx

```diff
@@ -0,0 +1,72 @@
+import { Typography } from "@mui/material";
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import EditIcon from '@mui/icons-material/Edit';
+import IconButton from '@mui/material/IconButton';
+import { useNavigate } from "react-router-dom";
+
+const EntriesList = ({ nodes, title }) => {
+    const navigate = useNavigate();
+
+    // Handler function for button clicks
+    const handleClick = (event, id) => {
+        event.preventDefault();
+        navigate('/entry/'+id);
+    }
+    return ( 
+        <div className="entry">
```

Comment:
please avoid using className, you could try `<Box sx={{}}>`. using hardcoded direct css classes tend to be difficult to manage  at some point :(